INTEGRATING.md

```diff
@@ -19,3 +19,28 @@ You can integrate it using the Prometheus Charm library. The documentation of th
 
 
 Besides this charm also provides the `loki_push_api` interface through the `log_proxy`, so can be used as a Log Proxy between a workload charm and a charm that also implements the `loki_push_api` interface, for instance [Loki](https://charmhub.io/loki-k8s). The documentation of the charm library, available through [Charmhub](https://charmhub.io/loki-k8s) provides further details.
+
+
+## Deployment scenarios
+```mermaid
+flowchart LR
+    subgraph COS Lite
+        prometheus
+        loki
+    end
+    subgraph K8s cluster
+        prometheus               ---      |remote_write| grafana-agent
+        loki                     ---     |loki_push_api| grafana-agent
+        grafana-agent            --- |prometheus_scrape| prometheus-scrape-config
+
+        grafana-agent            --- |prometheus_scrape| C1[Sidecar Charm 1]
+        C1                       ---     |loki_push_api| grafana-agent
+
+        prometheus-scrape-config --- |prometheus_scrape| C2[Sidecar Charm 2]
+        C2                       ---     |loki_push_api| grafana-agent
+    end
+
+    click prometheus https://github.com/canonical/prometheus-k8s-operator/
+    click loki https://github.com/canonical/loki-k8s-operator/
+    click prometheus-scrape-config https://github.com/canonical/prometheus-scrape-config-k8s-operator/
+```
```

Comment:
Nice!!
From my point of view,  I would remove the `prometheus-scrape-config` since it not in the scope of this doc.

I would add a text like the following before the diagram:

Let's say we have a K8s cluster running two applications packed as [Charmed Operators](https://juju.is/docs/sdk/charmed-operators) we need to observe with [COS Lite](https://charmhub.io/cos-lite):

- Sidecar charm 1
- Sidecar charm 2

Sending logs and metrics to `COS Lite` it's quite simple, we only need to deploy Grafana Agent Charmed Operator in the K8s cluster, and relate it with `Sidecar charm 1` and `Sidecar charm 2`